src/components/connected/mesInformations/ModalUpdateForm.js

```diff
@@ -36,8 +36,8 @@ function ModalUpdateForm({ form, showModal, setShowModal, formOrigin }) {
       }, conseiller?._id, user.name, password));
     } else if (formOrigin === 'informations') {
       dispatch(formInformationsActions.updateInformations({
```

Comment:
au cas où : est-ce qu'il y aurait un autre endroit qui corrige ce souci avec verifyFormulaire => https://sentry.incubateur.net/organizations/betagouv/issues/13456/?query=is%3Aunresolved&referrer=issue-stream

Reply:
Il se pourrait bien que ce soit un sentry que j'ai déjà corrigé, mais en attente de mise en prod !